### ISSUE TITLE
Make denomination field translatable

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/id-tagging-schema#254]
It would be useful to provide translations for non-English speakers

**Comment by 1ec5** [comment]
Should we split up the Denomination field into multiple fields, one per religion? That would allow us to filter out irrelevant values, which would be a significant usability improvement. On the other hand, the list of denominations for some religions is rather open-ended, and the less common denominations are the ones that benefit the most from a taginfo-based list.

**Comment by matkoniecz** [comment]
> Should we split up the Denomination field into multiple fields, one per religion? That would allow us to filter out irrelevant values, which would be a significant usability improvement. 

That would be also very useful for some other data consumers. For example in https://github.com/streetcomplete/StreetComplete/issues/1737 I would be able to limit to list of Christian denomination/rites/churches without maintaining own list which `denomination` values are Christian.

> On the other hand, the list of denominations for some religions is rather open-ended, and the less common denominations are the ones that benefit the most from a taginfo-based list.

Is it possible to show Christian denominations/rites/churches and then taginfo values which exclude known Muslim/etc denominations?

And for mosques show Muslim denominations/variants and then taginfo values which exclude Christian/etc denominations?

Even just showing all denominations specific to given religion and then all taginfo values would be significant usability improvement.

**Comment by 1ec5** [comment]
> Is it possible to show Christian denominations/rites/churches and then taginfo values which exclude known Muslim/etc denominations?
> 
> And for mosques show Muslim denominations/variants and then taginfo values which exclude Christian/etc denominations?

Not really, because the relationships between the religions and denominations isn’t defined anywhere in the tagging system. Only Wikidata has that information in machine-readable format.

We could start to curate denominations by creating a separate denomination field for each religion. Each field would have `type` set to `combo` and `autoSuggestions` set to `false`. Users would still be able to enter unlisted values, as long as `customValues` remains `false`, but we’d have to keep an eye on the most popular `denomination` values to make sure they’re getting surfaced in the dropdown as localizable suggestions.

**Comment by matkoniecz** [comment]
> Not really, because the relationships between the religions and denominations isn’t defined anywhere in the tagging system. Only Wikidata has that information in machine-readable format.

To clarify: I was proposing to start recording this info in this preset.

And OSM Wiki has already this classification for at least more popular ones

**Comment by tyrasd** [comment]
> Users would still be able to enter unlisted values, as long as `customValues` remains `false`

you mean as long as `customValues` remains `true`, right? :wink:
